### ISSUE TITLE
missing npm package for latest release

### ISSUE THREAD

**Issue by AdrienLemaire** [dequelabs/axe-core#2646]
The **@axe-core/react** npm package is still at v4.0.0, when 3 releases seem to have been done since.

![](https://i.imgur.com/6vedmz7.png)

```console
$ npm search @axe-core/react
NAME                      | DESCRIPTION          | AUTHOR          | DATE       | VERSION  | KEYWORDS
@axe-core/react           | Dynamic…             | =npmdeque…      | 2020-08-25 | 4.0.0    | accessibility a11y lint test
```

**Comment by straker** [comment]
Thanks for letting us know. We're planning on getting a release of all axe-core modules with the latest version of axe-core out soon.

**Comment by straker** [comment]
We just released all axe-core-npm (@axe-core) with the latest axe-core v4.1.1.

**Comment by AdrienLemaire** [comment]
![](https://i.imgur.com/uL57IOW.png)

**Comment by straker** [comment]
The versions of axe-core and @axe-core packages are not guaranteed to be the same. In this case, the version released of all @axe-core packages [is v4.1.0](https://www.npmjs.com/package/@axe-core/react/v/4.1.0), which uses [axe-core v4.1.1](https://github.com/dequelabs/axe-core-npm/blob/develop/packages/react/package.json#L51).
